index.js

```diff
@@ -233,12 +233,33 @@ var handleCloudWatch = function(event, context) {
   var region = event.Records[0].EventSubscriptionArn.split(":")[3];
   var subject = "AWS CloudWatch Notification";
   var alarmName = message.AlarmName;
-  var metricName = message.Trigger.MetricName;
+  var trigger = message.Trigger;
+  var alarmExpression;
+  if (typeof trigger.MetricName === "undefined") {
+    // This is a CloudWatch metric math alarm. Instead of a MetricName and
+    // statistic there is an array of Metrics where the first element is the
+    // math expression.
+
+    // first build a dictionary mapping metric ids to metric name and statistic
+    var sourceMetrics = {};
+    for (const metric of trigger.Metrics.slice(1)) {
```

Comment:
`const` and `for...of` loops are ES6, but the rest of this file seems to be using ES5 (e.g. `var` instead of `let` or `const`). I'd make sure ES6 is supported and that ES5 is not otherwise a requirement. It is odd to see both in one file.

As ES5, this would be:

```javascript
var triggerMetricsLength = trigger.Metrics.length;
for (var i = 1; i < triggerMetricsLength; i++) {
  var metric = trigger.Metrics[i];
  // ...
}
```

Reply:
combined this change with below recommendation